files/common/usr/bin/delphix-startup-screen

```diff
@@ -0,0 +1,400 @@
+#!/usr/bin/python3
+"""
+This program displays the delphix console screen which includes
```

Comment:
I believe this comment can be placed below the copyright comment.. IIRC, this is the convention we use in the SDB repo (docstring below the copyright).